packages/react/src/components/FluidSearch/FluidSearch.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright IBM Corp. 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+import Search from '../Search';
+import { usePrefix } from '../../internal/usePrefix';
+import { FormContext } from '../FluidForm/FormContext';
+
+const FluidSearch = React.forwardRef(function FluidSearch(
+  { className, ...other },
+  ref
+) {
+  const prefix = usePrefix();
+  const classNames = classnames(`${prefix}--search--fluid`, className);
+
+  return (
+    <FormContext.Provider value={{ isFluid: true }}>
+      <Search ref={ref} className={classNames} {...other} />
+    </FormContext.Provider>
+  );
+});
+
+FluidSearch.propTypes = {
+  /**
+   * Specify an optional value for the `autocomplete` property on the underlying
+   * `<input>`, defaults to "off"
+   */
+  autoComplete: PropTypes.string,
```

Comment:
Should we still provide the default within this component? I think without them we won't see the defaults in the storybook documentation and prop tables, etc.